config.go

```diff
@@ -80,9 +81,10 @@ func userConfigFinder() string {
 // GetStrict. It checks both $HOME/.ssh/config and /etc/ssh/ssh_config for keys,
 // and it will return parse errors (if any) instead of swallowing them.
 var DefaultUserSettings = &UserSettings{
-	IgnoreErrors:       false,
-	systemConfigFinder: systemConfigFinder,
-	userConfigFinder:   userConfigFinder,
+	IgnoreErrors:         false,
+	IgnoreMatchDirective: true,
```

Comment:
I'm thinking that default should be false -- do not ignore match directive. This way we cause no change to the current behaviour.

Reply:
Ah yes forgot to put it back was checking the unit were working